### PR TITLE
Add progress loggers for client

### PR DESCRIPTION
**Pull request by BogdanLivadariu**
when using the client to sign some big files, the console is pretty `quiet` about this.

in order to track the progress on where you're at with the signing, the console will be more than obvious on what the client is doing
screen shot 2017-10-31 at 15 04 28

### CODE REVIEW

**Comment by koenvd** [comment]
This would be really cool to have 👍 !

File: RemoteSignTool/RemoteSignTool.Client/Program.cs [hunk 1]
```diff
@@ -5,6 +5,7 @@
 using System.Linq;
 using System.Net.Http;
 using System.Net.Http.Headers;
+using System.Net.Http.Handlers;
```
Comment (leszekrogowski) [inline RemoteSignTool/RemoteSignTool.Client/Program.cs]: You can sort usings.

File: RemoteSignTool/RemoteSignTool.Client/Program.cs [hunk 2]
```diff
@@ -204,7 +206,13 @@ static int Main(string[] args)
 
         private static async Task<string> CommunicateWithServer(string archivePath, string signSubcommands)
         {
-            using (var client = new HttpClient())
+            var progressHandler = new ProgressMessageHandler();
```
Comment (leszekrogowski) [inline RemoteSignTool/RemoteSignTool.Client/Program.cs]: I'm not sure if it's completely necessary and correct but we can try to put ProgressMessageHandler in the using.

File: RemoteSignTool/RemoteSignTool.Client/Program.cs [hunk 3]
```diff
@@ -204,7 +206,13 @@ static int Main(string[] args)
...
+
+            progressHandler.HttpSendProgress += SendProgressHandler;
+            progressHandler.HttpReceiveProgress += ReceiveProgressHandler;
+
+
```
Comment (leszekrogowski) [inline RemoteSignTool/RemoteSignTool.Client/Program.cs]: Unnecessary return.

**Review by leszekrogowski — COMMENTED** [review]
In my opinion it looks good and it can be merged. You can consider my comments.

**Comment by BogdanLivadariu** [comment]
@leszekrogowski comments addressed

**Review by leszekrogowski — APPROVED** [review]

**Pull request merged**